packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -1,16 +1,20 @@
-import { Prisma } from "@prisma/client";
 // eslint-disable-next-line no-restricted-imports
```

Comment:
This file is the same as it was earlier before we reverted profile querying except for doing the normalization before sending the payload.